docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -0,0 +1,244 @@
+
+```js
+const transferOwnershipPayload =
+  web3.eth.abi.encodeFunctionSignature('transferOwnership(address)') +
+  web3.eth.abi.encodeParameter('address', newKeyManager._address).substring(2);
+
+await oldKeyManager.methods['execute(bytes)'](transferOwnershipPayload).send({
+  from: account.address,
+  gas: 1000000,
+  gasPrice: '1000000000',
+});
+```
+
+## Step 7 - Claim ownership of the Universal Profile
+
+Create a payload for `claimOwnership()` and claim the ownership of your Universal Profile from your new LSP6 Key Manager.
+
+```js
+const claimOwnershipPayload =
+  web3.eth.abi.encodeFunctionSignature('claimOwnership()');
+
+await newKeyManager.methods['execute(bytes)'](claimOwnershipPayload).send({
+  from: account.address,
+  gas: 1000000,
+  gasPrice: '1000000000',
+});
+```
+
+## Final code
+
+```javascript title="constants.js"
+export const PRIVATE_KEY = '0x...';
+export const universalProfileAddress = '0x...';
+export const keyManagerAddress = '0x...';
+```
+
+```javascript title="upgrade-lsp6.js"
+import LSP6KeyManager from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json' assert { type: 'json' };
+import {
+  PRIVATE_KEY,
+  keyManagerAddress,
+  universalProfileAddress,
+} from './constants.js';
+import Web3 from 'web3';
+const web3 = new Web3('https://rpc.l14.lukso.network');
+
+const upgradeLSP6 = async () => {
+  // Initialize the controller account
+  const account = web3.eth.accounts.privateKeyToAccount(PRIVATE_KEY);
+  web3.eth.accounts.wallet.add(account);
+
+  // Initialize your current LSP6 Key Manager
+  const oldKeyManager = new web3.eth.Contract(
+    LSP6KeyManager.abi,
+    keyManagerAddress,
+  );
+
+  // Deploy a new LSP6 Key Manager
+  const NewKeyManager = new web3.eth.Contract(LSP6KeyManager.abi);
+  const newKeyManager = await NewKeyManager.deploy({
+    data: LSP6KeyManager.bytecode,
+    arguments: [universalProfileAddress],
+  }).send({
+    from: account.address,
+    gas: 3000000,
+    gasPrice: '1000000000',
+  });
+
+  console.log(newKeyManager._address);
+
+  // Transfer the ownership of your Universal Profile from the current LSP6 Key Manager to a new LSP6 Key Manager
+  const transferOwnershipPayload =
+    web3.eth.abi.encodeFunctionSignature('transferOwnership(address)') +
+    web3.eth.abi
+      .encodeParameter('address', newKeyManager._address)
+      .substring(2);
+
+  await oldKeyManager.methods['execute(bytes)'](transferOwnershipPayload).send({
+    from: account.address,
+    gas: 1000000,
+    gasPrice: '1000000000',
+  });
+
+  // Accept the ownership of your Universal Profile from the new LSP6 Key Manager
+  const claimOwnershipPayload =
+    web3.eth.abi.encodeFunctionSignature('claimOwnership()');
+
+  await newKeyManager.methods['execute(bytes)'](claimOwnershipPayload).send({
+    from: account.address,
+    gas: 1000000,
+    gasPrice: '1000000000',
+  });
+};
+
+await upgradeLSP6();
+```
+
+## Testing the new LSP6 Key Manager
```

Comment:
As you always used simple present in other headings, i would keep this one in line too- removing the present progressive. I would suggest to always have a look at the headings on the right side and see if they align on the page- kinda hard to do this from plain code. It also helps with adapting the length of headings, so they look decent all together.
```suggestion
## Test the new LSP6 Key Manager
```